lib/ci-helper.ts

```diff
@@ -486,13 +486,13 @@ export class CIHelper {
      */
     public async handleComment(commentID: number): Promise<void> {
         const comment = await this.github.getPRComment(commentID);
-        const match = comment.body.match(/^(\/[-a-z]+)(\s+(.*))?$/);
+        const match = comment.body.match(/^\s*(\/[-a-z]+)(\s+(.*?))\s*$/);
```

Comment:
I think we still need the `?` after `(\s+(.*?))`, otherwise `/submit` would no longer be recognized...

Reply:
Sorry about that.  Forgot to retest with no trailing blanks after playing with third parameter support.

Reply:
No worries!